### ISSUE TITLE
1.0.3 Shift Key (selection) No Longer Works

### ISSUE THREAD

**Issue by NouGithub** [giuspen/cherrytree#2397]
**Version, Operative system**
Ubuntu 18.04

**Describe the bug**
Upgraded to 1.0.3 and the shift key for text selection no longer works.

**Comment by giuspen** [comment]
You must be hitting a keyboard shortcut as I did migrate the default Ctrl+Alt keyboard shortcuts to default Shift+Alt.
Go to the preferences dialog, tab keyboard shortcuts, and disable all the keyboard shortcuts that you don't use, or change removing the Shift, that should fix this.
If you understand what keyboard shortcut in particular is creating the issue let me know

**Comment by NouGithub** [comment]
Got it! It was Shift+Up, Shift+Right, etc... the shortcuts to move node positions.  Thank you for helping me sort that out so quickly! Phew!

(As a side note, I found the change of KB shortcuts quite problematic. Since I use shortcuts extensively and have been using Cherrytree for almost 10 years, adjusting to the new shortcuts has been quite challenging. Moreover, I had already customized many of the shortcuts, so reverting one of them to default cleared all of my custom shortcuts. I am gradually restoring them as I use them, but I would like to suggest **highlighting entries that have non-default or user-customized shortcuts**. Additionally, it would be helpful if there was an option to **export custom shortcuts**.)

Many thanks again!
